src/Sum/README.md

```diff
@@ -0,0 +1,147 @@
+# Sum
+```haskell
+Sum Number
+```
+`Sum` is a `Monoid` that will combine two numbers using Addition.
+
+```js
+const Sum = require('crocks/Sum')
+const mconcat = require('crocks/helpers/mconcat')
+const mconcatMap = require('crocks/helpers/mconcatMap')
+
+Sum(13)
+//=> Sum 13
+
+mconcat(Sum, [ 5, 2 ])
+//=> Sum 7
+
+Sum(10)
+  .concat(10)
+//=> Sum 20
+
+// mapToSumByTen :: [ Number ] -> Sum
+const sumByTen =
+  mconcatMap(Sum, x => x * 10)
+
+sumByTen([ 2, 2 ])
+//=> Sum 40
+```
+
+## Implements
+`Semigroup`, `Monoid`
+
+## Constructor Methods
+
+### empty
+```haskell
+Sum.empty :: () -> Sum
+```
+
+`empty` provides the identity for the `Monoid` in that when the value it
+provides is `concat`ed to any other value, it will return the other value. In
+the case of `Sum` the result of `empty` is `0`. `empty` is available on both
+the Constructor and the Instance for convenience.
+
+```js
+const Sum = require('crocks/Sum')
+
+Sum.empty()
+//=> Sum 0
+
+Sum.empty()
+  .concat(Sum.empty())
+//=> Sum 0
+
+Sum(4)
+  .concat(Sum.empty())
+//=> Sum 4
+```
+
+### type
+```haskell
+Sum.type :: () -> String
+```
+
+`type` provides a string representation of the type name for a given type in
+`crocks`. While it is used mostly internally for law validation, it can be
+useful to the end user for debugging and building out custom types based on the
+standard `crocks` types. While type comparisons can easily be done manually by
+calling `type` on a given type, using the `isSameType` function hides much of
+the boilerplate. `type` is available on both the Constructor and the Instance
+for convenience.
+
+```js
```

Comment:
Should probably also show an instance compare:

```javascript
const Prod = require('crocks/Prod')
const Sum = require('crocks/Sum')
const isSameType = require('crocks/predicates/isSameType')

const prod5 = Prod(5)

prod5.type()
//=> "Prod"

isSameType(Sum, prod5)
//=> false

isSameType(Prod, prod5)
//=> true

isSameType(Prod.empty(), prod5)
//=> true
```

Reply:
I'll do a `isSameType(Sum.empty(), sum5)` for Sum, and `isSameType(Prod.empty(), prod5)` for Prod?

Reply:
yeppers